### ISSUE TITLE
[TikTok] Most video metadata is missing.

### ISSUE THREAD

**Issue by Jadoo4QFan** [yt-dlp/yt-dlp#4923]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I remove or skip any mandatory\* field

### Checklist

- [X] I'm requesting a site-specific feature
- [X] I've verified that I'm running yt-dlp version **2022.09.01** ([update instructions](https://github.com/yt-dlp/yt-dlp#update)) or later (specify commit)
- [X] I've checked that all provided URLs are playable in a browser with the same IP and same login details
- [X] I've searched the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)
- [X] I've read about [sharing account credentials](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#are-you-willing-to-share-account-details-if-needed) and I'm willing to share it if required

### Region

California, USA 

### Example URLs

https://www.tiktok.com/t/ZT
R5twdVJ/ 

### Provide a description that is worded well enough to be understood

The TikTok info json accessed via the API contains way more information about the video than the info json. As an archivist, this disappoints me, as I need all metadata to archive mutiple TikToks. After this and the HD video problem are fixed I will start using this service primarily to download tiktok videos.

### Provide verbose output that clearly demonstrates the problem

- [X] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [X] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
[Documents]$ yt-dlp https://www.tiktok.com/t/ZT
R5twdVJ/ -vU -j --write-pages 
[debug] Command-line config: ['https://www.tiktok.com/t/ZTR5twdVJ/', '-vU', '-j', '--write-pages']
[debug] Encodings: locale UTF-8, fs utf-8, pref UTF-8, out utf-8, error utf-8, screen utf-8
[debug] yt-dlp version 2022.09.01 [5d7c7d6] (pip)
[debug] Python 3.9.10+ (CPython 64bit) - macOS-15.4.1-iPhone14,5-arm-64bit 
[debug] Checking exe version: ffmpeg -bsfs
[debug] Checking exe version: ffprobe -bsfs
[debug] exe versions: ffmpeg n5.0.1-4-ga5ebb3d25e (setts), ffprobe n5.0.1-4-ga5ebb3d25e, phantomjs present, rtmpdump present
[debug] Optional libraries: Crypto-3.15.0, certifi-2022.06.15, mutagen-1.45.1, sqlite3-2.6.0, websockets-10.3
[debug] Proxy map: {}
[debug] Loaded 1670 extractors
[debug] Fetching release info: https://api.github.com/repos/yt-dlp/yt-dlp/releases/latest
Latest version: 2022.09.01, Current version: 2022.09.01
yt-dlp is up to date (2022.09.01)
[debug] [generic] Extracting URL: https://www.tiktok.com/t/ZTR5twdVJ/
[generic] ZTR5twdVJ: Downloading webpage
[redirect] Following redirect to https://www.tiktok.com/@newlinersseries/video/7122853180995620142?_t=8VfStdoFy70&_r=1
[TikTok] Setting up session
[debug] [TikTok] Extracting URL: https://www.tiktok.com/@newlinersseries/video/7122853180995620142?_t=8VfStdoFy70&_r=1
[TikTok] 7122853180995620142: Downloading video details
[TikTok] Saving request to 7122853180995620142_https_-_api-h2.tiktokv.com_aweme_v1_aweme_detail_aweme_id=7122853180995620142_version_name=20.9.3_version_code=293_build_number=20.9.3_manifest_version_code=293_update_version_code=293___8cbd97dc648a5bbd0096ffa09ce56bcb.dump
[debug] Sort order given by extractor: quality, codec, size, br
[debug] Formats sorted by: hasvid, ie_pref, quality, vcodec, acodec, filesize, fs_approx, tbr, vbr, abr, lang, res, fps, hdr:12(7), channels, asr, proto, vext, aext, hasaud, source, id
[debug] Default format spec: bestvideo*+bestaudio/best
[info] 7122853180995620142: Downloading 1 format(s): bytevc1_720p_657943-2
{"id": "7122853180995620142", "extractor_key": "TikTok", "extractor": "TikTok", "webpage_url": "https://www.tiktok.com/@7106641030017270830/video/7122853180995620142", "title": "Jadoo 4 (Jadoo4Q) #jadootv #androidkitkat #jadoo4 #jadoo4q #androidtv #androidtvbox #meme #techtok #android #shitpost", "description": "Jadoo 4 (Jadoo4Q) #jadootv #androidkitkat #jadoo4 #jadoo4q #androidtv #androidtvbox #meme #techtok #android #shitpost", "view_count": 9342, "like_count": 547, "repost_count": 1, "comment_count": 21, "uploader": "newlinersseries", "creator": "newlinersseries", "uploader_id": "7106641030017270830", "uploader_url": "https://www.tiktok.com/@MS4wLjABAAAAWRTxa9TmYNrI8T-5s9uJDRAEjUK47kceEtCRyFUIEsMang6XJwIvZMYGqCObcv8g", "track": "Sound", "album": null, "artist": "Stevey Obesey", "timestamp": 1658418494, "formats": [{"url": "https://api-h2.tiktokv.com/aweme/v1/play/?video_id=v12044gd0000cbcn95bc77ua7eavlpd0&line=0&watermark=1&logo_name=tiktok_m&source=AWEME_DETAIL&file_id=c4e109faee884548820660360bfb71c0", "filesize": 1719198, "ext": "mp4", "acodec": "aac", "source_preference": -2, "format_id": "download_addr-0", "format_note": "Download video, watermarked (API)", "vcodec": "h264", "width": 576, "height": 1024, "preference": -2, "protocol": "https", "video_ext": "mp4", "audio_ext": "none", "format": "download_addr-0 - 576x1024 (Download video, watermarked (API))", "resolution": "576x1024", "dynamic_range": "SDR", "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.18 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate", "Cookie": "odin_tt=57bf67f280d97a9f9a5a0a6dfb205402a184d46abebb013cb6bb6f0148a935ae124a43f46f2d0bdd10a0b63c1fcf37cde50cb614ae4256d95e23861ab9598bd79703aaaa533bc5bf63f86c67d189c4d3; msToken=n0mBzJj-KgqH0FrofoAcXRuBD0_AryG-cxFYJQD48cVmykptc3sYDuBrKRQY_9bIN8-XNIHVwFVyN-8ug-KZGXo99sPgVqapXXZ_UzCwyWDTDsPiQ4HHEQ=="}}, {"url": "https://v19.tiktokcdn.com/a4266bd4e25f92b5541aceaedccd714c/6321b733/video/tos/maliva/tos-maliva-ve-0068c799-us/0f17a7687e014b3e9e76e972e2673a69/?a=1180&ch=0&cr=3&dr=0&lr=all&cd=0%7C0%7C0%7C3&cv=1&br=2390&bt=1195&cs=0&ds=3&ft=FEedZWD00Q12BS0rzwqUx_pna2HC~5Agf~hIn&mime_type=video_mp4&qs=0&rc=MzRlZTkzOTU7ODtoZDw0N0BpM3N5aGQ6ZjhxZTMzZzczNEBgNGMtMDMtNTExNWJjLWJgYSNhaV40cjRfNmBgLS1kMS9zcw%3D%3D&l=2022091405124001024513620906096185&btag=80000&cc=4", "filesize": 1719198, "ext": "mp4", "acodec": "aac", "source_preference": -1, "format_id": "download_addr-1", "format_note": "Download video, watermarked", "vcodec": "h264", "width": 576, "height": 1024, "preference": -2, "protocol": "https", "video_ext": "mp4", "audio_ext": "none", "format": "download_addr-1 - 576x1024 (Download video, watermarked)", "resolution": "576x1024", "dynamic_range": "SDR", "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.18 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate"}}, {"url": "https://v25.tiktokcdn.com/61b176fc34fb177a74dc6ffdb230f4a4/6321b733/video/tos/maliva/tos-maliva-ve-0068c799-us/0f17a7687e014b3e9e76e972e2673a69/?a=1180&ch=0&cr=3&dr=0&lr=all&cd=0%7C0%7C0%7C3&cv=1&br=2390&bt=1195&cs=0&ds=3&ft=FEedZWD00Q12BS0rzwqUx_pna2HC~5Agf~hIn&mime_type=video_mp4&qs=0&rc=MzRlZTkzOTU7ODtoZDw0N0BpM3N5aGQ6ZjhxZTMzZzczNEBgNGMtMDMtNTExNWJjLWJgYSNhaV40cjRfNmBgLS1kMS9zcw%3D%3D&l=2022091405124001024513620906096185&btag=80000&cc=e", "filesize": 1719198, "ext": "mp4", "acodec": "aac", "source_preference": -1, "format_id": "download_addr-2", "format_note": "Download video, watermarked", "vcodec": "h264", "width": 576, "height": 1024, "preference": -2, "protocol": "https", "video_ext": "mp4", "audio_ext": "none", "format": "download_addr-2 - 576x1024 (Download video, watermarked)", "resolution": "576x1024", "dynamic_range": "SDR", "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.18 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate"}}, {"url": "https://api-h2.tiktokv.com/aweme/v1/play/?video_id=v12044gd0000cbcn95bc77ua7eavlpd0&line=0&is_play_url=1&source=PackSourceEnum_AWEME_DETAIL&file_id=5e4a241ff4d449409173c55db789df13", "filesize": 1689683, "ext": "mp4", "acodec": "aac", "source_preference": -2, "format_id": "h264_540p_1203049-0", "format_note": "Direct video (API)", "vcodec": "h264", "width": 576, "height": 1024, "tbr": 1203, "quality": 1, "protocol": "https", "video_ext": "mp4", "audio_ext": "none", "vbr": 1203, "abr": 0, "format": "h264_540p_1203049-0 - 576x1024 (Direct video (API))", "resolution": "576x1024", "dynamic_range": "SDR", "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.18 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate", "Cookie": "odin_tt=57bf67f280d97a9f9a5a0a6dfb205402a184d46abebb013cb6bb6f0148a935ae124a43f46f2d0bdd10a0b63c1fcf37cde50cb614ae4256d95e23861ab9598bd79703aaaa533bc5bf63f86c67d189c4d3; msToken=n0mBzJj-KgqH0FrofoAcXRuBD0_AryG-cxFYJQD48cVmykptc3sYDuBrKRQY_9bIN8-XNIHVwFVyN-8ug-KZGXo99sPgVqapXXZ_UzCwyWDTDsPiQ4HHEQ=="}}, {"url": "https://v19.tiktokcdn.com/0024271b89a4711e87906202a8811201/6321b733/video/tos/maliva/tos-maliva-ve-0068c799-us/3faec4ffcf904cb299abd98087bc5174/?a=1180&ch=0&cr=3&dr=0&lr=all&cd=0%7C0%7C0%7C3&cv=1&br=2348&bt=1174&cs=0&ds=6&ft=FEedZWD00Q12BS0rzwqUx_pna2HC~5Agf~hIn&mime_type=video_mp4&qs=0&rc=Nmk8Omc4NjQzPDc3aTdkaEBpM3N5aGQ6ZjhxZTMzZzczNEAuYTVfMmA0NjExYWMuLzEyYSNhaV40cjRfNmBgLS1kMS9zcw%3D%3D&l=2022091405124001024513620906096185&btag=80000&cc=4", "filesize": 1689683, "ext": "mp4", "acodec": "aac", "source_preference": -1, "format_id": "h264_540p_1203049-1", "format_note": "Direct video", "vcodec": "h264", "width": 576, "height": 1024, "tbr": 1203, "quality": 1, "protocol": "https", "video_ext": "mp4", "audio_ext": "none", "vbr": 1203, "abr": 0, "format": "h264_540p_1203049-1 - 576x1024 (Direct video)", "resolution": "576x1024", "dynamic_range": "SDR", "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/96.0.4664.18 Safari/537.36", "Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8", "Accept-Language": "en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate"}}, {"url": "https://v25.tiktokcdn.com/264dec2acbcca489173a09fee3b04cf4/6321b733/video/tos/maliva/tos-maliva-ve-0068c799-us/Chrome/96.0.4664.18ast5-i-0068-tx/98aa582453d244ba84abeca19280d221.image?x-expires=1663153200&x-signature=dQKghG1G3LlojugtGkOkeL1Anoc%3D&s=AWEME_DETAIL&se=false&sh=&sc=dynamic_cover&l=2022091405124001024513620906096185"}, {"id": "ai_dynamic_cover_bak", "url": "https://p16-sign.tiktokcdn-us.com/tos-useast5-p-0068-tx/b5e1c61d89a24cad81423425b6ce9a96_1658418495~tplv-dmt-logom:tos-useast5-i-0068-tx/98aa582453d244ba84abeca19280d221.image?x-expires=1663153200&x-signature=3bTojfIBJzl3vv2qSouUxGCPmu4%3D&s=AWEME_DETAIL&se=false&sh=&sc=dynamic_cover&l=2022091405124001024513620906096185"}, {"id": "ai_dynamic_cover_bak", "url": "https://p19-sign.tiktokcdn-us.com/tos-useast5-p-0068-tx/b5e1c61d89a24cad81423425b6ce9a96_1658418495~tplv-dmt-logom:tos-useast5-i-0068-tx/98aa582453d244ba84abeca19280d221.image?x-expires=1663153200&x-signature=dQKghG1G3LlojugtGkOkeL1Anoc%3D&s=AWEME_DETAIL&se=false&sh=&sc=dynamic_cover&l=2022091405124001024513620906096185"}, {"id": "animated_cover", "url": "https://p16-sign.tiktokcdn-us.com/tos-useast5-p-0068-tx/b5e1c61d89a24cad81423425b6ce9a96_1658418495~tplv-dmt-logom:tos-useast5-i-0068-tx/98aa582453d244ba84abeca19280d221.image?x-expires=1663153200&x-signature=3bTojfIBJzl3vv2qSouUxGCPmu4%3D&s=AWEME_DETAIL&se=false&sh=&sc=dynamic_cover&l=2022091405124001024513620906096185"}, {"id": "animated_cover", "url": "https://p19-sign.tiktokcdn-us.com/tos-useast5-p-0068-tx/b5e1c61d89a24cad81423425b6ce9a96_1658418495~tplv-dmt-logom:tos-useast5-i-0068-tx/98aa582453d244ba84abeca19280d221.image?x-expires=1663153200&x-signature=dQKghG1G3LlojugtGkOkeL1Anoc%3D&s=AWEME_DETAIL&se=false&sh=&sc=dynamic_cover&l=2022091405124001024513620906096185"}, {"id": "cover", "url": "https://p16-sign.tiktokcdn-us.com/tos-useast5-p-0068-tx/a419388328344675a445a9e4039a4cca~tplv-dmt-logoccm:300:400:tos-useast5-i-0068-tx/98aa582453d244ba84abeca19280d221.jpeg?x-expires=1663153200&x-signature=qJt2XAiQSvb3ufMfD%2B%2BkSmWyVwM%3D&s=AWEME_DETAIL&se=false&sh=&sc=cover&l=2022091405124001024513620906096185"}, {"id": "cover", "url": "https://p16-sign.tiktokcdn-us.com/tos-useast5-p-0068-tx/a419388328344675a445a9e4039a4cca~tplv-dmt-logoccm:300:400:tos-useast5-i-0068-tx/98aa582453d244ba84abeca19280d221.webp?x-expires=1663153200&x-signature=uYl18pQAre5dp%2BA829eUWf6cIVw%3D&s=AWEME_DETAIL&se=false&sh=&sc=cover&l=2022091405124001024513620906096185"}, {"id": "cover", "url": "https://p19-sign.tiktokcdn-us.com/tos-useast5-p-0068-tx/a419388328344675a445a9e4039a4cca~tplv-dmt-logoccm:300:400:tos-useast5-i-0068-tx/98aa582453d244ba84abeca19280d221.webp?x-expires=1663153200&x-signature=rjhZMTnArurku%2F%2BcCd5Xwton2eg%3D&s=AWEME_DETAIL&se=false&sh=&sc=cover&l=2022091405124001024513620906096185"}, {"id": "dynamic_cover", "url": "https://p16-sign.tiktokcdn-us.com/tos-useast5-p-0068-tx/01c15cf0b3bd41a090f3f5a6ea059184_1658418495~tplv-dmt-logom:tos-useast5-i-0068-tx/98aa582453d244ba84abeca19280d221.image?x-expires=1663153200&x-signature=ud9zdr6MpvAGaEi%2BbaI9vxHXQ%2F8%3D&s=AWEME_DETAIL&se=false&sh=&sc=dynamic_cover&l=2022091405124001024513620906096185"}, {"id": "dynamic_cover", "url": "https://p19-sign.tiktokcdn-us.com/tos-useast5-p-0068-tx/01c15cf0b3bd41a090f3f5a6ea059184_1658418495~tplv-dmt-logom:tos-useast5-i-0068-tx/98aa582453d244ba84abeca19280d221.image?x-expires=1663153200&x-signature=DQM74tWrYVCVic4B7EJX4lXCrLs%3D&s=AWEME_DETAIL&se=false&sh=&sc=dynamic_cover&l=2022091405124001024513620906096185"}, {"id": "origin_cover", "url": "https://p16-sign.tiktokcdn-us.com/tos-useast5-p-0068-tx/66c91179605b45a6ae99aadfcc640755_1658418495~tplv-tiktokx-360p.jpeg?x-expires=1663153200&x-signature=WigT5QVYzTRsm5i70CXv2rewLRg%3D&s=AWEME_DETAIL&se=false&sh=&sc=feed_cover&l=2022091405124001024513620906096185"}, {"id": "origin_cover", "url": "https://p16-sign.tiktokcdn-us.com/tos-useast5-p-0068-tx/66c91179605b45a6ae99aadfcc640755_1658418495~tplv-tiktokx-360p.webp?x-expires=1663153200&x-signature=PaC02Yg%2BI7RN%2FjJ%2BNXRHFG64H%2FI%3D&s=AWEME_DETAIL&se=false&sh=&sc=feed_cover&l=2022091405124001024513620906096185"}, {"id": "origin_cover", "url": "https://p19-sign.tiktokcdn-us.com/tos-useast5-p-0068-tx/66c91179605b45a6ae99aadfcc640755_1658418495~tplv-tiktokx-360p.webp?x-expires=1663153200&x-signature=YrIVDBSuZUkxsPMt5QFefiPJ04Y%3D&s=AWEME_DETAIL&se=false&sh=&sc=feed_cover&l=2022091405124001024513620906096185"}], "duration": 11, "availability": null, "original_url": "https://www.tiktok.com/t/ZTR5twdVJ/", "webpage_url_basename": "7122853180995620142", "webpage_url_domain": "tiktok.com", "playlist": null, "playlist_index": null, "thumbnail": "https://p19-sign.tiktokcdn-us.com/tos-useast5-p-0068-tx/66c91179605b45a6ae99aadfcc640755_1658418495~tplv-tiktokx-360p.webp?x-expires=1663153200&x-signature=YrIVDBSuZUkxsPMt5QFefiPJ04Y%3D&s=AWEME_DETAIL&se=false&sh=&sc=feed_cover&l=2022091405124001024513620906096185", "display_id": "7122853180995620142", "fulltitle": "Jadoo 4 (Jadoo4Q) #jadootv #androidkitkat #jadoo4 #jadoo4q #androidtv #androidtvbox #meme #techtok #android #shitpost", "duration_string": "11", "upload_date": "20220721", "requested_subtitles": null, "_has_drm": null, "url": "https://v25.tiktokcdn.com/e004d74ee9685ed1d69cf46e9bb8bf68/6321b733/video/tos/maliva/tos-maliva-ve-0068c799-us/75be0b6032994c6c904a09b28b3656bf/?a=1180&ch=0&cr=3&dr=0&lr=all&cd=0%7C0%7C0%7C3&cv=1&br=1284&bt=642&cs=2&ds=3&ft=FEedZWD00Q12BS0rzwqUx_pna2HC~5Agf~hIn&mime_type=video_mp4&qs=14&rc=PDUzNTNoPGU1O2c1aGY2OkBpM3N5aGQ6ZjhxZTMzZzczNEAyMTE2NDBjXzAxMjMvXjRgYSNhaV40cjRfNmBgLS1kMS9zcw%3D%3D&l=2022091405124001024513620906096185&btag=80000&cc=e", "filesize": 924082, "ext": "mp4", "acodec": "aac", "source_preference": -1, "format_id": "bytevc1_720p_657943-2", "format_note": "Playback video", "tbr": 657, "vcodec": "h265", "fps": null, "height": null, "quality": 2, "width": null, "protocol": "https", "video_ext": "mp4", "audio_ext": "none", "vbr": 657, "abr": 0, "format": "bytevc1_720p_657943-2 - unknown (Playback video)", "resolution": null, "dynamic_range": "SDR", "http_headers": {"User-Agent": "Mozilla/5.0 (Windows NT 10

o
.0; Win64; x64) AppleWebKit/537.36 (KHTML, like
 Gecko) Chrome/96.0.4664.18 Safari/537.36", "Ac
cept": "text/html,application/xhtml+xml,applica
tion/xml;q=0.9,*/*;q=0.8", "Accept-Language": "
en-us,en;q=0.5", "Sec-Fetch-Mode": "navigate"},
 "epoch": 1663132361, "_filename": "Jadoo 4 (Ja
doo4Q) #jadootv #androidkitkat #jadoo4 #jadoo4q
 #androidtv #androidtvbox #meme #techtok #andro
id #shitpost [7122853180995620142].mp4", "filen
ame": "Jadoo 4 (Jadoo4Q) #jadootv #androidkitka
t #jadoo4 #jadoo4q #androidtv #androidtvbox #me
me #techtok #android #shitpost [712285318099562
0142].mp4", "urls": "https://v25.tiktokcdn.com/
e004d74ee9685ed1d69cf46e9bb8bf68/6321b733/video
/tos/maliva/tos-maliva-ve-0068c799-us/75be0b603
2994c6c904a09b28b3656bf/?a=1180&ch=0&cr=3&dr=0&
lr=all&cd=0%7C0%7C0%7C3&cv=1&br=1284&bt=642&cs=
2&ds=3&ft=FEedZWD00Q12BS0rzwqUx_pna2HC~5Agf~hIn
&mime_type=video_mp4&qs=14&rc=PDUzNTNoPGU1O2c1a
GY2OkBpM3N5aGQ6ZjhxZTMzZzczNEAyMTE2NDBjXzAxMjMv
XjRgYSNhaV40cjRfNmBgLS1kMS9zcw%3D%3D&l=20220914
05124001024513620906096185&btag=80000&cc=e", "_
type": "video", "_version": {"version": "2022.0
9.01", "current_git_head": null, "release_git_h
ead": "5d7c7d6", "repository": "yt-dlp/yt-dlp"}
}
```

**Comment by Jadoo4QFan** [comment]
Sorry I had to truncate the info json due to GitHub being a dick.

**Comment by Jadoo4QFan** [comment]
https://archive.org/download/7122853180995620142-https-api-h-2.tiktokv.com-aweme-v-1-aweme-detail-aweme-id-71/7122853180995620142_https_-_api-h2.tiktokv.com_aweme_v1_aweme_detail_aweme_id%3D7122853180995620142_version_name%3D20.9.3_version_code%3D293_build_number%3D20.9.3_manifest_version_code%3D293_update_version_code%3D293___8cbd97dc648a5bbd0096ffa09ce56bcb.dump

The TikTok original info json

**Comment by bashonly** [comment]
Tiktok changed something today and broke the extractor's API functionality. This will be fixed by #4920

**Comment by pukkandan** [comment]
Your requirement is too ambiguous. Tell us exactly which metadata fields you are expecting. Some of this will be fixed by https://github.com/yt-dlp/yt-dlp/pull/4920; but without more info nobody can say whether that solves your full issue

**Comment by Jadoo4QFan** [comment]
> Your requirement is too ambiguous. Tell us exactly which metadata fields you are expecting. Some of this will be fixed by #4920; but without more info nobody can say whether that solves your full issue

See the json I sent you. So we have 2 options.
1. The Red Pill: Just slam the entire aweme_detail as raw-metadata, full-tiktok-metadata, or aweme-detail. The easiest option and most data retentive option, however not that practical.
2. The Blue Pill: Nest most fields with metadata into the original info json, and rename most of them.

**Comment by bashonly** [comment]
>     1. The Red Pill: Just slam the entire aweme_detail as raw-metadata, full-tiktok-metadata, or aweme-detail. The easiest option and most data retentive option, however not that practical.

isn't this what `--write-pages` is for?

**Comment by pukkandan** [comment]
I am not asking you what you think the solution should be. I am asking exactly what metadata fields you are expecting to get

**Comment by Your-New-SJW-Waifu** [comment]
> Your requirement is too ambiguous. Tell us exactly which metadata fields you are expecting. Some of this will be fixed by #4920; but without more info nobody can say whether that solves your full issue

Ever since this became an issue I haven't been able to get the full resolution 720p video and have instead have only been able to only get the 540p video. It's more than just savings metadata. There seems to also be an issue with getting the full resolution playlist.

**Comment by Jadoo4QFan** [comment]
> > Your requirement is too ambiguous. Tell us exactly which metadata fields you are expecting. Some of this will be fixed by #4920; but without more info nobody can say whether that solves your full issue
> 
> Ever since this became an issue I haven't been able to get the full resolution 720p video and have instead have only been able to only get the 540p video. It's more than just savings metadata. There seems to also be an issue with getting the full resolution playlist.

Most of the fields from the dump file don’t get put in the TikTok info json, which is problematic for archivists.

**Comment by Jadoo4QFan** [comment]
> > > Your requirement is too ambiguous. Tell us exactly which metadata fields you are expecting. Some of this will be fixed by #4920; but without more info nobody can say whether that solves your full issue
> > 
> > 
> > Ever since this became an issue I haven't been able to get the full resolution 720p video and have instead have only been able to only get the 540p video. It's more than just savings metadata. There seems to also be an issue with getting the full resolution playlist.
> 
> Most of the fields from the dump file don’t get put in the TikTok info json, which is problematic for archivists.

Check the dumped json out. It contains way more information about the TikTok.

**Comment by Jadoo4QFan** [comment]
Never mind, I’ll add it myself.
